bin/qserv-check-integration.py

```diff
@@ -143,21 +148,26 @@ def _parse_args():
     else:
         args.mode_list = [args.mode]
 
+    if args.multi_node == True and args.mode != 'qserv':
+        _LOG.error("Select --mode=qserv for multi-node test")
+        sys.exit()
```

Comment:
`sys.exit()` without argument means success, add non-zero return code to return an error: `sys.exit(2)`